scripts/next-data/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,47 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
+
+import { getRelativePath } from './_helpers.mjs';
+
+const __dirname = getRelativePath(import.meta.url);
+const jsonFilePath = join(
+  __dirname,
+  '../../public/static/node-releases-data.json'
+);
+
+const generateNodeReleasesData = async () => {
+  const nodevuOutput = await nodevu();
+
+  const nodeReleases = new Map();
+  Object.values(nodevuOutput)
+    .reverse()
```

Comment:
I see you reverse the array here and down there. Is there any need of reversing the array, then? Since you're reversing it twice

Reply:
`nodevu`'s output contains duplicate `v0.x` versions. First reversal is for us to traverse the list increasing so we can pick the last version of each major. Second reversal is because we are displaying the list decreasing (from latest release).

Reply:
I think it's [intentional](https://github.com/cutenode/nodevu/blob/main/core/index.js#L24)
It seems there's a historical context about `v0.x` versions which I am not sure of... And this is consistent with [schedule.json](https://github.com/nodejs/Release/blob/main/schedule.json#L2-L13) too

After thinking again... Maybe I'll just filter out the duplicate `v0.x` versions using hardcoded `if` since this case is also hardcoded in `nodevu`. Not to mention that there won't be new `v0.x` version.

Something like this:
```js
nodeReleases.filter(release => release.major != 0 || release.version === '0.12.18');
```

Reply:
Sounds like good to me. The comment explaining why can point to nodevu's source :)